meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -43,13 +47,14 @@ do_install:append() {
     # Install domu-set-root script
     install -d ${D}${libdir}/xen/bin
     install -m 0744 ${WORKDIR}/domu-set-root ${D}${libdir}/xen/bin
-    install -d ${D}${sysconfdir}/systemd/system/domu.service.d
```

Comment:
This is not related to device trees. Please move those changes into a separate commit

Reply:
Agree.
Done.